test/jdk/javax/swing/JTabbedPane/7024235/Test7024235.java

```diff
@@ -81,6 +79,7 @@ public void run() {
             JFrame frame = new JFrame();
```

Comment:
I guess the frame needs to be disposed in finally block.

Reply:
It is already disposed of at line #90 but only for automated mode, if the test run manually the frame will not be disposed of + an exception is also will not throw.